### PR TITLE
readAloud-valence-jess

### PR DESCRIPTION
**Pull request by jessb0t**
@georgebuzzell

Please see latest comments in #4.

Looking forward to your feedback both on readAloud-valence and also on improvements to the brainBox workflow!

### CODE REVIEW

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (georgebuzzell) [inline brainy-ideas/readAloud-valence-jess/readme.md:74]: Things like this will be good to get Angela's input on :) No need to finalize the measures in brainbox though;

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (georgebuzzell) [inline brainy-ideas/readAloud-valence-jess/readme.md]: I do not recall them off the top of my head, but there are two that Angela reccomends. Warning though: these are typically pretty long to assess (like 30 mins plus) (if I remember correctly)

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (georgebuzzell) [inline brainy-ideas/readAloud-valence-jess/readme.md]: but only if it is free, i think it might not be...

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (georgebuzzell) [inline brainy-ideas/readAloud-valence-jess/readme.md]: I really think we need to dig into this idea of what to do during the blocks further. I am very hesitant about the "chat" idea, as this will lead to lots of noise in the data. Yes, you want ecological validity, but baby steps... The study design as a whole is already much, much more ecologically valid. Also, it is important to keep in mind that if you require chatting during the break, not everyone is an extrovert and actually likes to do so. I really think we should have something more controlled between the breaks, or even just nothing at all. But, I think we can discuss this more in the project repo, I dont think this needs to be hashed out here in brain box.

Comment (georgebuzzell) [reply brainy-ideas/readAloud-valence-jess/readme.md]: meta comment: we should provide some language in the brainbox readme to provide guidelines for what state a project has to be in to be "approved". Certainly not all the little experimental details. But there needs to be the basic structure, which directly tests well worded hypotheses, and there needs to be initial work put into operationalizing ivs/dvs. Basically the state this project is now at (which is great) :)

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (georgebuzzell) [inline brainy-ideas/readAloud-valence-jess/readme.md:91]: This is going to be something to discuss further. still not clear what they hypotheses are, etc. But ok for brainbox.

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (georgebuzzell) [inline brainy-ideas/readAloud-valence-jess/readme.md:96]: the whole study? I think i misunderstand, as you were also proposing chatting in between blocks.

**Review by georgebuzzell — APPROVED** [review]
Looks good, Jess. Of course, still lots of details to hash out, but I think the project has a fairly clear direction and that you should move ahead with creating repos.

One MAJOR comment is that I almost certainly think you need to include one of the more traditional lab tasks on which you are basing the eco valid study on. Otherwise, it is an extremely risky study to run, because if you find no effects, it will not ve clear whether that is because even the basic lab tasks effects were not replicable or whether it was do to the increased noise of the eco valid task. As a general rule in science, you never want to run a study where you could be left with a set of uninterpretable results. As currently set up, this study could end up that way. The standard approach when doing a more novel task as you are doing here, is to also include a more traditional task. That way, if you find nothing in your novel task, but do find something in the traditional task, you can reasonably assume that the basic effect of interest previously reported in the literature is real, at least using a traditional task, but that something about the novel task implementation obscured the effects. Alternatively, if you find nothing in your novel task, AND your traditional task does not replicate prior findings (spoiler alert: lots of things dont replicate) then you have reason to question the basic effect itself that was previously reported in the literature as opposed to your novel task per se.

your point is well taken that the focus is on this novel task, and you dont want to tire individuals out with another long task before hand. However, there are several options: 1) you shorten the traditional task and your novel task; 2) you counterbalance order of task completion (to measure and control for fatigue effects; 3) You always have participants do your novel task first and only do the traditional task second (but only if you are fairly certain the fatigue will not totally wipe out the effect).

i think most likely we want to go with option 3, because the great point you made about the traditioanl tasks making the flipping explicit, and so you dont want them to do that first.

so, in sum, I am aproving your project to move forward with the caveat that you will also need to include one of these traditional tasks that you are basing your novel study on. You could borrow one of the exact tasks, or we could create a similar one. We could shorten it, etc. And, we will almost certainly have participants perform the traditional task at the end of the study, after everything else is done. 

Please note that this approach I am advocating is the same approach used for all three fo the current studies. 

In sum, as long as you agree with adding a traditional task, in some form, tbd outside of brainbox, then please go ahead and move forward with creating the repos. However, if we are not yet in agreement here yet (and that is totally fine) then we should do another round of review in brainbox, focusing specifically on the traditional task piece. 

Again, great work, Jess! Very clever and interesting study you have put together; I look forward to seeing it fully come to life!!

**Review by georgebuzzell — APPROVED** [review]

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (jessb0t) [reply brainy-ideas/readAloud-valence-jess/readme.md]: And it's true that we could likely use education level here as an indicator of adequate reading ability.  Or include a practice paragraph that has a test question to gauge engagement with what is being read?

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (jessb0t) [reply brainy-ideas/readAloud-valence-jess/readme.md]: How can we tell?  Pearson has a paid version, but there are plenty of free versions online.  Not sure what counts as "public domain" here...

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (jessb0t) [reply brainy-ideas/readAloud-valence-jess/readme.md]: Do you have any ideas for what would serve as a good, neutral break?  I've been wracking my brain, but haven't come up with anything that feels appropriate.

RE: meta comment: I agree that we should have some indication of an appropriate state before approval.  In our case, I'd like to really get the main components of the task planned: what lexical decision task we're doing, how we're doing breaks, etc.  Then all the finer details (like specifics of the stimuli, etc.) can be hashed out later.  So, for now, I'm not going to merge.

Also, my own meta comment: Is the plan, when we do merge, that this folder lives permanently in brainBox, we close the brainBo issue, and the conversation picks back up over in the new project repo?

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (jessb0t) [reply brainy-ideas/readAloud-valence-jess/readme.md:91]: Are you referring to the breaks?  If so, ok to change this (see question above about ideas).

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (jessb0t) [reply brainy-ideas/readAloud-valence-jess/readme.md:96]: Haha, yes, the participant was supposed to just chat with their screen.  As in, the screen gave them a prompt and they could just speak, randomly, for a minute or so.  But I'm fine to do something different as a "break," just need some guidance!

**Comment by jessb0t** [comment]
@georgebuzzell 
Please see notes above in response to your comments.  I have updated the readme to include the lexical decision task, but haven't yet made other changes until we discuss further.

For the lexical decision task, based on my reading to date, we could attempt a replication of either Kousta et al. (2009) or Kazanas & Altarriba (2015), both of which published their stimuli.  I have outlined the pertinent factors below:

### (Kousta et al., 2009)
- standard lexical decision task (LDT)
- participants were native speakers of English in London
- stimuli are provided in published paper: 40 negative, 40 positive, and 40 neutral words that were matched triplet-wise along the lexical and sublexical dimensions listed here:
image
- nonce word distractors are **not** provided in paper: 40 negative, 40 positive, and 40 neutral words, which were used to create non-words by changing a single letter in a random position
- method is described clearly:
> Participants were tested individually. Each trial began with a fixation cross presented at the middle of the screen for 400 ms, followed by the presentation of the string for 2000 ms or until a response was given (whichever was earlier). Participants were instructed to respond as quickly and accurately as possible using the j key on the keyboard (marked green) when the string was a word and the f key (marked red) when the string was not a word. After response or time-out, there was a blank inter-trial interval of 1000 ms. After ten practice items, the 240 words and non-words were presented in a random order. The experimental session lasted approximately 12 min.
- this was replicated by Yap & Seow (2013) for both a standard LDT and for a go/no-go LDT (Singaporean English speakers); they extended their analysis to a distributional analysis of response times, but they had to create their own nonce words as these are not provided in the original paper
- **MAJOR ADVANTAGE**: this appears to be a foundational paper in work relating to valence and lexical decision (411 citations on Google Scholar)
- **MAJOR DISADVANTAGE**: since this was a standard LDT, it did not find a positivity advantage

### (Kazanas & Altarriba, 2015)
- primed lexical decision task
- participants were English speakers in the United States
- stimuli are provided in published paper: 112 experimental trials (half with real targets, half with nonce targets), equally divided among emotion word type (emotion v. emotion-laden) and valence (positive v. negative) blocks: 28 trials in each block, no words repeated throughout the trials, blocked by valence.  (Note: one of the matching dimensions for this set is markedly different than Kousta et al.: stimuli were matched on standard LDT response time.)<br/>
*NOTE*: I have a nagging concern about the stimuli because they imply that "bliss" and "happy" are unrelated semantically, which seems unlikely.  The stimuli set has been printed this way in at least two of their papers, in different journals.  The [LLAMA Lab](https://belemglopez.com/llamalab/) actually recently contacted the authors for clarification on whether this was a misprint and is waiting to hear back.
- nonce word distractors are **not** provided in paper: "Prime-target word pairs were created with equal numbers of related and unrelated pairs. Non-word pairs were created by changing one vowel in the target word, though each non-word was still pronounceable according to English phonological rules. To reduce any response bias, half of the word pairs included non-words."
- method is described clearly:
> Participants were presented with a fixation ‘+’ at the center of the computer screen for 500 ms. After this duration, a prime word (e.g., passion) appeared on the screen for 1000 ms, which participants read to themselves, silently. Then, the target word (e.g., love) appeared and they had a maximum of 2000 ms to respond, until the next trial began. Each participant was encouraged to respond as quickly and as accurately as possible by pressing the ‘m’ key if the target was a word and the ‘z’ key if the target was a non-word. Feedback was given according to the accuracy of each trial.
- another paper by Kazanas and Altarriba, also published in 2015, is *almost* a replication, except it uses a masked paradigm and participants were US-based Spanish/English bilinguals 
- **MAJOR ADVANTAGE**: these are the authors who have found the positivity advantage when you hold standard LDT RT constant for the stimuli in a primed paradigm involving emotion and emotion-laden words
- **MAJOR DISADVANTAGE**: additional coding complexity in the task setup and I have an outstanding question about the accuracy of the stimuli as published

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (jessb0t) [reply brainy-ideas/readAloud-valence-jess/readme.md]: Decided to go with relying upon education level and included challenge questions after passage readings, which will enable us to identify participants who were either insufficiently engaged in the texts or unable to fully understand them.

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (jessb0t) [reply brainy-ideas/readAloud-valence-jess/readme.md]: I adjusted the break idea to be more neutral.

For our meta-discussion, we still need to determine:

- guidelines for what constitutes an adequate level for approval to "graduate" from brainBox (although this may also be something that you want to maintain at your own discretion, as you might want, say, a strong undergrad to plan further than a fifth year PhD?).
- What remains in brainBox after approval and merge of the idea branch.  I would now suggest that the project lead move all the materials over to the new repo and leave, in the subdirectory on brainBox, a simple readme that indicates the last commit on brainBox and date of transfer to a separate repo.  I think this will keep brainBox clean, with just a single folder (containing only a readme) for all "graduated" ideas, but enable track back through the brainBox history.

Let me know your thoughts on these and I will use them to guide my readAloud-valence merge here, as well as update brainBox guidance itself!

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (jessb0t) [reply brainy-ideas/readAloud-valence-jess/readme.md:91]: Modified breaks.

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (jessb0t) [reply brainy-ideas/readAloud-valence-jess/readme.md:96]: Modified breaks.

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (jessb0t) [reply brainy-ideas/readAloud-valence-jess/readme.md]: Or perhaps we leave the main readme that outlines all the study ideas instead?

**Comment by jessb0t** [comment]
After further research, I have determined that Kousta et al. (2009) will be the better replication.  Although Kousta et al. (and the Yap + Seow replication) did not find a positivity advantage, a mega-analysis of >12k words by Kuperman et al. (2014) found that valence interacts with word frequency, which is a plausible explanation for Kousta et al.'s findings given that they did not control for this interaction in their analysis.

In order to perform a true replication, however, additional information is needed from Dr. Vigliocco (the corresponding author for the Kousta et al (2009) paper), if she were amenable to providing it:
1. For the frequency measure, did you use KF or HAL from the ELP?
2. Would you share your nonce word stimuli for a true replication?
3. Would you share your practice terms for a true replication?
4. Would you be willing to share your extended version of ANEW for our analysis purposes?
5. Is it intentional that "victory" is duplicated in the published list of word stimuli?  If not, is this a typo or was the word inadvertently included twice in the experiment?
6. Were participants given any feedback concerning accuracy or response latency?
7. Was stimulus presentation jittered?

Importantly, I believe that [surprisal](https://en.wikipedia.org/wiki/Information_content) is going to be a very important variable in our analyses.  We will need the assistance of a computational linguist to determine surprisal measures for the final stimuli.  Some researchers who work in this area include Marten van Schijndel (https://github.com/vansky) and Cory Shain (https://github.com/coryshain/)?  (Cory Shain is doing some cool work on continuous-time deconvolutional regressive neural networks ([CDRNN](https://aclanthology.org/2021.acl-long.288.pdf)), with a 12-minute introduction [here](https://screencast-o-matic.com/watch/cr13ocV13bR).  That is far too advanced by this study's needs, but I thought @georgebuzzell  would find it interesting!)  The surprisal measures will be needed for analysis, so we have some time to figure out a game plan for NLP assistance.

File: brainy-ideas/readAloud-valence-jess/readme.md
Comment (georgebuzzell) [reply brainy-ideas/readAloud-valence-jess/readme.md]: people post copies all the time online. but you have to check the license to see if you are actually supposed to use it or not. There are plenty of alternatives to BDI anyways, E.g., PHQ

**Comment by georgebuzzell** [comment]
It is hard to tell what has changed since the last round of review, bc this is the same pr. However, maybe there is an easy way to see the changes you made, without re-reading things. 

in general, though, i do think the level of detail I'd prefer for brainbox was passed a bit ago. That is, if the idea is for there to be multiple of these going on at once, amongst everything else, I will have trouble tracking them. I think we want to limit the scope/goal of brainbox to be a place for the main idea of the study to take shape, enough to where I commit to moving forward with it.

Re the replication, I don't think we need to do an exact replication per se, I only meant that we want to have at least a conceptual replication built-in, something likely to show an effect, in case the novel effects you are testing do not pan out. That way, we can have some level of confidence that the null effects are not due to a wider, systematic issue.
-I am trying to recall now, though, the context of my initial suggestion for embedding a replication. Was the idea to have participants engage in a more traditional task (i.e. one of the two you mention)? Or was suggesting more of a conceptual replication of a prior effect, but within the context of the natural text reading?

Surprisal seems like something that would be great to quantify and look at!

For what to do with the stuff on brainbox after merging, if it is all within a subdirectory for this proposal, I would prefer to keep it all here. Yes, copy over pieces that are needed to the new repo, but I see no reason to remove things from here, as they document the development of the project. But maybe I miss why this will clutter things?

I have not preference for Kousta vs the other study in terms of a replication, but please see my comment above re replication.
-pending whether we are running a seperate, traditional lab task in the form of Kousta, yes, you might as well reach out for the stimuli and other details. but, no stress if they dont provide them and we need to fill in.
-but again, even if we run the seperate kousta task, I feel like I might have been suggesting to try to embed a replication within the context of the natural text reading. Though, we could do both. Also, I might be mis-remembering....

**Comment by georgebuzzell** [comment]
Just to clarify: The plan is to first run a behavioral version of the study, then redo with EEG if the behavioral study pans out, yes?

I think this is all set for creating a repo for this project. :)

**Review by georgebuzzell — APPROVED** [review]

**Comment by georgebuzzell** [comment]
@jessb0t This draft is left open while you work to refine brainbox and/or set up the repo, yes? Just checking to make sure nothing further is needed here from me :)

Awesome project proposal; I look forward to seeing it in action!

**Comment by jessb0t** [comment]
@georgebuzzell Confirming!  I was intending to set it up and actually show the merge live during the lab meeting. :nerd_face:

**Pull request merged**
